src/EnergyPlus/ScheduleManager.hh

```diff
@@ -541,10 +547,14 @@ struct ScheduleManagerData : BaseGlobalStruct
         scheduleTypes.clear(); // Allowed Schedule Types
         scheduleTypeMap.clear();
 
-        for (int i = 0; i < (int)schedules.size(); ++i)
+        // Don't try to delete the constant schedules which were not malloced
+        for (int i = 2; i < (int)schedules.size(); ++i) {
             delete schedules[i];
+        }
         schedules.clear(); // Schedule Storage
         scheduleMap.clear();
+        // Put back the ConstantSchedules
+        AddConstantSchedulesToContainers();
```

Comment:
In clear_state, I avoid trying to delete the stack objects. And I re-add them back in schedules/scheduleMap, so that it's back to the init_initial_state state.